### ISSUE TITLE
Neodash is sending "sentry" stuff from my self-hosted docker image

### ISSUE THREAD

**Issue by diyoyo** [neo4j-labs/neodash#616]
Hello, 

- NeoDash version: v2.3.4
- Neo4j Database version: 5.11.0 (Community).

- **Steps to reproduce**
- Just use NeoDash and perform actions
- In self-hosted mode, it sends a bunch of analytics on the outside world...
-  Can I disable this or should I just let Brave continue to do the blocking?
Thanks

Additionally, include (as appropriate) screenshots, drawings, etc.
<img width="703" alt="image" src="https://github.com/neo4j-labs/neodash/assets/6624639/409d43aa-9332-4596-90d3-6f13f7ad7659">

**Comment by BennuFire** [comment]
Hi @diyoyo!

That's weird, it should only apply for the public neodash deploy. I'll take a look on that. For the record, Sentry helps us to gather error logs in a secure way (without any sensitive data, actually, no text data at all). This in order to quickly determine bugs that may be affecting a lot of people without them coming here to visit us 😉

Thanks!

**Comment by diyoyo** [comment]
Hi @BennuFire ,
I trust you are doing it for the right reasons, it just started to make me worry about all these other dockers I am running out of the box without even thinking about uncontrolled analytics. Anyways, it's a good thing since it raised my awareness on the topic.
It also pushed me beyond my limits and I took a look at the source. Since it seems to result in a 1-pager app, I just had to comment out the sentry part in the index file and rebuild the docker image as explained in the doc.
Next step: me learning how to add a parameter in Dockerfile to be able to activate Sentry dynamically when calling `docker compose` :)
